frontend/src/edit.jsx

```diff
@@ -83,93 +89,133 @@ export default function EditMenu(props) {
+        </HTMLSelect>
+      </div>
+      <Switch
+        checked={patternSamplingMode}
+        labelElement={<strong>Create new pattern</strong>}
+        onChange={togglePatternSamplingMode}
+      />
+      {patternSamplingMode ? patternSampleForm : <div />}
     </>
   );
 
   return (
     <div className="menu-container">
-      <Tabs id="Tabs" onChange={handleTabChange} selectedTabId={activeTab}>
+      <Tabs
+        id="Tabs"
+        onChange={(tabID) => setActiveTab(tabID)}
+        selectedTabId={activeTab}
+      >
         <Tab
           id="create"
           title="Create"
           panel={(
             <div>
               <ParcelTable parcelTable={parcelTable} />
               <br />
-              <form onSubmit={handleSubmitNew}>
-                <RadioGroup
-                  inline={false}
-                  label="Modify the landuse of this parcel by selecting a pattern:"
-                  onChange={handleRadio}
-                  selectedValue={pattern}
-                >
-                  {patterns.map(pattern => <Radio key={pattern} value={pattern} label={pattern} />)}
-                </RadioGroup>
-                <h4>Add this modification to a scenario</h4>
-                <datalist id="scenariolist">
-                  {Object.values(savedScenarios).map(item => <option key={item} value={item} />)}
-                </datalist>
-                <input
-                  type="search"
-                  id="scenarioName"
-                  list="scenariolist"
-                  value={scenarioName}
-                  onChange={(event) => setScenarioName(event.currentTarget.value)}
-                />
-                <button type="submit">Submit</button>
-              </form>
+              {
+                (parcel)
+                  ? (
+                    <form onSubmit={handleSubmitNew}>
+                      <RadioGroup
+                        className="sidebar-subheading"
+                        inline
+                        label="Modify the landuse of this parcel by:"
+                        onChange={(event) => setConversionOption(event.target.value)}
+                        selectedValue={conversionOption}
+                      >
+                        <Radio key="wallpaper" value="wallpaper" label="wallpaper" />
+                        <Radio key="paint" value="paint" label="paint" />
+                      </RadioGroup>
+                      <div className="conversion-panel">
+                        {
+                          (conversionOption === 'paint')
+                            ? (
+                              <HTMLSelect
+                                onChange={(event) => setSingleLULC(event.target.value)}
+                              >
+                                {Object.entries(lulcCodes)
+                                  .map(([k, v]) => <option key={k} value={k}>{v}</option>)}
```

Comment:
It could be nice, and consistent with other "paint" interfaces, to include a little color swatch square to the left of each LULC option in the dropdown. I'm not sure how difficult that would be, though.

Reply:
Great idea, I'm sure that's doable. Maybe we should wait and see just how many landuse codes there are in the ultimate LULC from Eric & team. If it really is dozens/hundreds, we may need to re-design the dropdown altogether.

Reply:
Good point, we can hold off on that.